tests/core/daemon/test_daemon.py

```diff
@@ -809,6 +810,57 @@ async def test_bad_json(daemon_connection_and_temp_keychain: Tuple[aiohttp.Clien
     assert message["data"]["error"].startswith("Expecting property name")
 
 
+@pytest.mark.parametrize(
+    "method, parameter, response_data_dict",
```

Comment:
I'm not clear why any of this is any better? It seems all the same to me.

Reply:
It groups the id with the data instead of separated in a parallel list.  It puts names to the values instead of, again, parallel lists between the data and the parameter names.  it provides a single distinct type to reference which results in all values being typed and checked.  Probably more, but maybe that's enough.

Reply:
Used a dataclasss and consolidated some tests into a larger parameterized test